sycl/test-e2e/FilterSelector/select_device_hip.cpp

```diff
@@ -0,0 +1,62 @@
+// RUN: %{build} -o %t.out
+// RUN: env ONEAPI_DEVICE_SELECTOR=hip:gpu %t.out
+//
+// Checks if only specified device types can be acquired from select_device
+// when ONEAPI_DEVICE_SELECTOR is set.
+// Checks that no device is selected when no device of desired type is
+// available.
+//
+// REQUIRES: hip,gpu
```

Comment:
HIP assumes gpu already.
```suggestion
// REQUIRES: hip
```